lib/browse_everything/driver/s3.rb

```diff
@@ -6,6 +6,8 @@ class S3 < Base
       DEFAULTS = { signed_url: true, region: 'us-east-1' }.freeze
       CONFIG_KEYS = [:app_key, :app_secret, :bucket].freeze
 
+      attr_reader :file_entries
```

Comment:
or bucket_entries, but yes, something that means "everything that is in this path"